src/MentionableTextarea.vue

```diff
@@ -147,6 +150,9 @@
                 const uniqueId = uuid();
                 const newId = `${id}-${uniqueId}`;
                 this.elementIdMap.set(id, newId);
+                if (id === 'UL'){
+                    this.selectedULID = newId;
```

Comment:
We actually don't need this. We can achieve the same behaviour with `this.elementIdMap.get('UL')`. What do you think?

Reply:
`elementIdMap` is the first time I've heard of it. I'll look into it and update the code.

Reply:
Using elementIdMap was the correct way. Fixed it!